modules/effects/src/main/scala/dev/profunktor/redis4cats/interpreter/Redis.scala

```diff
@@ -113,8 +113,8 @@ object Redis {
     Resource.make(acquire)(release).map(_.asInstanceOf[RedisCommands[F, K, V]])
   }
 
-  def masterSlave[F[_]: Concurrent: ContextShift: Log, K, V](
-      conn: RedisMasterSlaveConnection[K, V]
+  def masterReplica[F[_]: Concurrent: ContextShift: Log, K, V](
+      conn: RedisMasterReplicaConnection[K, V]
   ): F[RedisCommands[F, K, V]] =
     new Redis[F, K, V](new RedisStatefulConnection(conn.underlying)).asInstanceOf[RedisCommands[F, K, V]].pure[F]
```

Comment:
This `asInstanceOf` could probably be replaced with an upcast, `: RedisCommands[...]`

Reply:
Even better, I can use `widen` :)

Reply:
Neat.